README.md

```diff
@@ -46,18 +46,18 @@ Developers can include the Java AMQP 0-9-1 Client using the following maven depe
 
 # Running the WebSocket demo
 
-## From the command-line
+## Run From the Command-Line
 0. Change directory: ```cd ws/demo/target```
```

Comment:
The hyphen is there as a description of "interface." The wikipedia entry you linked refers to "command lines" (no hyphen, without the "interface" object).